<?xml version="1.0" encoding="UTF-8"?>
<class xmlns="http://xml.phpdox.net/src" full="Symfony\Component\HttpKernel\Exception\FatalErrorException" namespace="Symfony\Component\HttpKernel\Exception" name="FatalErrorException" abstract="false" final="false" start="23" end="25">
  <file path="/home/web/example/src/freegame_developer/vendor/symfony/debug/Exception" file="FatalErrorException.php" realpath="/home/web/example/src/freegame_developer/vendor/symfony/debug/Exception/FatalErrorException.php" size="3340" time="2016-09-06T07:26:07+00:00" unixtime="1473146767" sha1="993b8ed713956bd1175838e8220a7f5c7c35aeba" relative="freegame_developer/vendor/symfony/debug/Exception/FatalErrorException.php"/>
  <docblock>
    <description compact="Fatal Error Exception."/>
    <author value="Fabien Potencier &lt;fabien@symfony.com&gt;"/>
    <author value="Konstanton Myakshin &lt;koc-dp@yandex.ru&gt;"/>
    <author value="Nicolas Grekas &lt;p@tchwork.com&gt;"/>
    <deprecated value="Deprecated in 2.3, to be removed in 3.0. Use the same class from the Debug component instead."/>
  </docblock>
  <extends full="ErrorException" namespace="" name="ErrorException"/>
</class>
